gamePage.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <link href="style.css" rel="stylesheet" type="text/css">
+    <script src="script.js" defer></script>
+    <title>Pro-To-Type Typing Speed Game</title>
+</head>
+
+<body>
+    <main>
+        <div class="linkToRules">
+            <a href="index.html">Back to rules</a>
+        </div>
+        <h2>Copy the text below: </h2>
+        <div id="textsToTypeContainer">
```

Comment:
this div has a scrollbar that makes it look slightly off center and it looks a bit awkward:
![image](https://user-images.githubusercontent.com/25844355/220406473-249061a8-6330-478a-9336-113aa2a0ca4b.png)

you can [remove the scrollbar while still preserving its functionality](https://blog.hubspot.com/website/hide-scrollbar-css):
![image](https://user-images.githubusercontent.com/25844355/220406393-eed4e685-2830-45d0-b524-f50dbfa44163.png)